docs/book/src/providers/implementers-guide/building_running_and_testing.md

```diff
@@ -0,0 +1,131 @@
+# Building, Running, Testing
+
+## Docker Image Name
+
+The patch in `config/default/manager_image_patch.yaml` will be applied to the manager pod. 
+Right now there is a placeholder `IMAGE_URL`, which you will need to change to your actual image.
+
+### Development Images
+It's likely that you will want one location and tag for release development, and another during development.
+
+The approach most Cluster API projects is using [a `Makefile` that uses `sed` to replace the image URL][sed] on demand during development.
```

Comment:
This line doesn't render correctly (see https://deploy-preview-1682--kubernetes-sigs-cluster-api.netlify.com/providers/implementers-guide/building_running_and_testing.html)